### ISSUE TITLE
Identify a check by name within CI action

### ISSUE THREAD

**Issue by boesing** [laminas/laminas-ci-matrix-action#76]
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes
| RFC         | yes
| BC Break    | no

#### Summary

Hey there,

I am currently working on something and I have the following use-case:
- run tests on all supported PHP versions (works out-of-the-box)
- run tests on lowest PHP version with locked deps (works ootb)
- run tests on lowest PHP version with locked deps via `additional_checks` (works ootb, **but**...)

So the latter does work due to prior changes made to this action. The problem is, that I now want to identify this exact `additional_check` somehow. Because the main idea of this check would be:

Run tests on lowest PHP version with locked deps **but** disable one or more PHP extensions (to see if tests are still passing due to precautions or polyfills) and/or disallow PHP functions.

So, I'd like to have some kind of optional `name` within the `job` configuration which can be used as some kind of unique identifier. When added, this can be extract within a `pre-run.sh` for example.
An alternative would be to actually allow disabling extensions via the `job` array. That would make the whole thing easier but maybe its worth having both "features" up and running.

WDYT?

**Comment by Ocramius** [comment]
Makes sense. Additionally, it would be a good occasion for shortening default names a bit (they all overflow GitHub 's UI)

**Comment by boesing** [comment]
Yah, but we have to be careful here as these names might be used with "exclude". 

https://github.com/boesing/psalm-plugin-stringf/blob/586ae0569b3ef5bcef1035dd70d4778ce8a03bce/.laminas-ci.json#L13

Because thats the only way how you can exclude jobs as of now. But maybe this is only relevant for this action and not for the container action. 👍🏻

**Comment by boesing** [comment]
Oh, I actually found out that the checks do actually have a name. It is just not used anywhere:
https://github.com/laminas/laminas-ci-matrix-action#providing-specific-checks-to-run

I'll see if we can somehow pass these around while also slightly modifying the `exclude` logic so it can both exclude job names **and** job command strings.
